app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -99,6 +100,11 @@ class Config {
     return ServiceAccountInfo.fromJson(json.decode(rawValue));
   }
 
+  Future<ServiceAccountCredentials> get taskLogServiceAccount async {
+    final String rawValue = await _getSingleValue('TaskLogServiceAccount');
```

Comment:
Why not use existing DevicelabServiceAccount, and just grant logging.write to it?

Reply:
My understanding of service accounts is that in a perfect world they should have the minimum permissions needed. I didn't want to start a pattern of adding a bunch of responsibilities to one service account which would make it hard to track if there are any unnecessary permissions granted to the account.

If we remove this logging functionality down the line, it makes it easy to just delete this service account with this feature than to figure out if the permissions are used elsewhere.